### ISSUE TITLE
Poor keyboard navigation support on assignments page

### ISSUE THREAD

**Issue by bluelhf** [Aalto-LeTech/aplus-courses#1257]
#### Describe the bug
The default focus element for the Assignments page is the search bar, and when the search bar is focused keyboard navigation can't be used to open exercise rounds or make submissions.

#### To Reproduce
1. Open the A+ Courses page
2. Use <kbd>Alt + Right</kbd> and <kbd>Alt + Left</kbd> to switch to Modules and then back to Assignments
3. Notice that the search bar is focused
4. Attempt to use the arrow keys, spacebar or Enter key to navigate the assignment tree
5. Notice you can switch between rounds but not expand them nor can you make submissions on expanded assignments.

#### Expected behavior
It should be possible to use <kbd>Tab</kbd> to navigate to the assignments tree, or the assignments page should switch focus to the assignments tree when keyboard navigation (such as arrow keys or Enter) is used.

#### Screenshots
<img width="365" height="160" alt="The assignments page with the search bar focused and selected assignment round highlighted but not focused" src="https://github.com/user-attachments/assets/09cf9582-cabd-4228-be80-6bb7663fc134" />

Note that when the UI is in the state shown in the screenshot, it's impossible to expand any of the assignment tree elements (`Kierros X`), only navigate between them using the Up and Down arrow keys.

#### Environment (please complete the following information):
 - Operating System: Linux, Devuan 7 (Ceres) on X11 7.7.
 - IntelliJ IDEA version (found from **Help > About**): IntelliJ IDEA 2025.2.3 (Ultimate Edition)
 - Version of A+ Courses plugin (found from **A+ > About**): 4.4.0

**Comment by bluelhf** [comment]
I'd be happy to look later tonight :)
